.ci/jenkins_job_build.sh

```diff
@@ -88,20 +97,19 @@ then
 else
 	# Othewise we test an specific branch
 	# GIT_BRANCH env variable is set by the jenkins Github Plugin.
-	remote="${GIT_BRANCH/\/*/}"
-	branch="${GIT_BRANCH/*\//}"
+	export branch="${GIT_BRANCH/*\//}"
 
 	if [ "${kata_repo}" != "${tests_repo}" ]
 	then
 		# Use the correct branch for testing.
 		# 'tests' repository branch should have the same name
 		# as the kata repository branch that will be tested.
 		pushd "${test_repo_dir}"
-		git fetch "$remote" && git checkout "$branch"
+		git fetch origin && git checkout "$branch"
```

Comment:
I am ok with the this change but could you tell me why remove the remote usage?

Reply:
As we do not have other remotes other than `origin`, I prefer to change it, wdyt?

Reply:
I am fine with it :)